result/isislsp.yaml

```diff
@@ -0,0 +1,341 @@
+            When set, the originator supports partition repair.
+          type: boolean
+        attached_error:
+          description: |-
+            When set, the originator is attached to another area using the referred metric.
+          type: boolean
+        attached_expense:
+          description: |-
+            When set, the originator is attached to another
+            area using the referred metric.
+          type: boolean
+        attached_delay:
+          description: |-
+            When set, the originator is attached to another
+            area using the referred metric.
+          type: boolean
+        attached_default:
+          description: |-
+            When set, the originator is attached to another
+            area using the referred metric.
+          type: boolean
+        overload:
+          description: |-
+            When set, the originator is overloaded, and must
+            be avoided in path calculation.
+          type: boolean
+
+    IsisLsp.IsReachability:
+      description: >-
+        This container describes list of ISIS neighbors and attributes.
+      type: object
+      properties:
+        neighbors: 
+          description: >-
+            This container describes IS neighbors.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.neighbor'
+
+    IsisLsp.ExtendedIsReachability:
+      description: >-
+        This container describes list of ISIS neighbors and attributes.
+      type: object
+      properties:
+        neighbors: 
+          description: >-
+            This container describes IS neighbors.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.neighbor'
+
+    IsisLsp.neighbor:
+      description: >-
+        This container describes IS neighbors.
+      type: object
+      properties:
+        system_id:
+          description: >-
+            The System ID for this emulated ISIS router, e.g. "640100010000".
+          type: string
+          format: hex
+
+    IsisLsp.Ipv4InternalReachability:
+      description: |-
+        This container defines list of IPv4 internal reachability information.
+      type: object
+      properties:
+        prefixes:
+          description: >-
+            This container describes IS neighbors.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.V4Prefix'
+
+    IsisLsp.Ipv4ExternalReachability:
+      description: |-
+        This container defines list of IPv4 external reachability information.
+      type: object
+      properties:
+        prefixes:
+          description: >-
+            This container describes IS neighbors.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.V4Prefix'
+
+    IsisLsp.V4Prefix:
+      description: >-
+        This group defines attributes of an IPv4 standard prefix.
+      type: object
+      properties:
+        ipv4_address:
```

Comment:
Should this property have 'format : ipv4' in it?

Reply:
agrred.